pipeline/notifications/check_message.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+
+import prefect
+from prefect import task
+
+from pipeline.types import Notification, User
+
+
+class Notifier:
+    def __init__(self):
+        pass
+
+    def check_msg_notifier(self, notif: Notification) -> List[User]:
```

Comment:
We may want to rename this method to better describe what it does. Since it returns users, we could call it `get_users_to_notify()`.

You might want to add another method (or combine into this method) something like `get_notification()` which returns all the information needed to actually send the notification, like email, subject, etc..

What information will another engineer need to write a new notification implementation for one of the other use cases we want to support? Will these method names still be relevant to them? When designing a parent and child classes, these are topics to think about, so that the parent class helps clarify the logic of the child classes (implementations), rather than forcing the child classes to conform to a structure that does not make sense for them.